google/cloud/firestore_v1/async_collection.py

```diff
@@ -130,17 +127,26 @@ async def list_documents(
         async for i in iterator:
             yield _item_to_document_ref(self, i)
 
-    async def get(
-        self, transaction=None
-    ) -> AsyncGenerator[async_document.DocumentSnapshot, Any]:
-        """Deprecated alias for :meth:`stream`."""
-        warnings.warn(
-            "'Collection.get' is deprecated:  please use 'Collection.stream' instead.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        async for d in self.stream(transaction=transaction):
-            yield d  # pytype: disable=name-error
+    async def get(self, transaction=None) -> list:
+        """Read the documents in this collection.
+
+        This sends a ``RunQuery`` RPC and returns a list of documents
```

Comment:
So, this deprecated bit makes me wonder if we shouldn't include this. Is that perhaps why it isn't here?

Reply:
See following comment.